autoria.py

```diff
@@ -0,0 +1,242 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def make_request(url: str) -> json:
+    """Send get request and return data in JSON.
+
+    Args:
+        url - string with url returning needed
+        data (categories, bodystyles etc.)
+
+    Returns:
+        JSON-formatted response text.
+    """
+    response = requests.get(url)
+    if response.status_code == 200:
+        return json.loads(response.text)
+    else:
+        print('Ups! Something went wrong in processing of {}'.format(url))
+
+
+def get_categories() -> json:
+    """Get available vehicle types from auto.ria.com.
+
+    No args needed because the list has fixed url.
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Легковые",
+            "value": 1
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories'
+    return make_request(url)
+
+
+def get_bodystyles(category: int) -> json:
+    """Get available bodystyles from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
```

Comment:
```
category - category id to get data for.
```

We know that it's integer from the type hint (so it is not necessary to repeat it) and obviously, it is the category we need.